Class-19-(Authentication).md

```diff
@@ -0,0 +1,25 @@
+# Class 19: Authentication
+
+## In the prep work this week, the students learned
+
+1. What authentication is.
+1. What sessions and cookies are and how sessions and cookies are used to verify a user's identity.
+1. The importance of encryption.
+1. Why developers use one-way encryption to protect users' passwords.
+1. How to use Spring Security in an application to enable authentication.
```

Comment:
We aren't actually using Spring Security in this lesson. We should say something like "How to use request filtering in an application..."

Reply:
Fixed!